backend/runner/taskcheck/statement_affected_rows_report_executor.go

```diff
@@ -99,8 +99,16 @@ func (s *StatementAffectedRowsReportExecutor) Run(ctx context.Context, _ *store.
 	switch instance.Engine {
 	case db.Postgres:
 		return reportStatementAffectedRowsForPostgres(ctx, sqlDB, renderedStatement)
-	case db.MySQL:
-		return reportStatementAffectedRowsForMySQL(ctx, sqlDB, renderedStatement, dbSchema.Metadata.CharacterSet, dbSchema.Metadata.Collation)
+	case db.MySQL, db.MariaDB, db.OceanBase:
+		return reportStatementAffectedRowsForMySQL(ctx, instance.Engine, sqlDB, renderedStatement, dbSchema.Metadata.CharacterSet, dbSchema.Metadata.Collation)
+	case db.TiDB:
+		return []api.TaskCheckResult{{
+			Status:    api.TaskCheckStatusSuccess,
+			Namespace: api.BBNamespace,
+			Code:      common.Ok.Int(),
+			Title:     "OK",
+			Content:   "0",
+		}}, nil
```

Comment:
Why we skip tidb here?

Reply:
Thanks!

Actually `IsTaskCheckReportSupported` was incorrect, we only supported MySQL and Postgres before. WIth your changes, I think the list should be MySQL, Postgres, MariaDB and OceanBase.

I suggest we remove TiDB from `IsTaskCheckReportSupported` and related codes.

Reply:
Done.